### PR TITLE
Desired states to fix start screen for continous playback

### PR DESCRIPTION
**Pull request by wunnath**

### CODE REVIEW

File: sdk/react/ooyalaSkinPanelRenderer.js [hunk 1]
```diff
@@ -275,7 +275,11 @@ OoyalaSkinPanelRenderer.prototype.renderScreen = function(overlayType, ad, scree
 
   switch (screenType) {
     case SCREEN_TYPES.START_SCREEN: 
-      return this.renderStartScreen(); 
+      if(this.skin.state.desiredstate !="desired_play") {
+      return this.renderStartScreen();
```
Comment (lental) [inline sdk/react/ooyalaSkinPanelRenderer.js]: tabbing

**Comment by lental** [comment]
You need to set the initial state for the skin.  see index.android.js, and index.ios.js

**Comment by lental** [comment]
as a note, iOS is not affected, because as long as "this.skin.state.desiredState" is unchanged, then we will get the old, always-renderStartScreen behavior

File: sdk/react/ooyalaSkinPanelRenderer.js [hunk 2]
```diff
@@ -275,7 +275,11 @@ OoyalaSkinPanelRenderer.prototype.renderScreen = function(overlayType, ad, scree
 
   switch (screenType) {
     case SCREEN_TYPES.START_SCREEN: 
-      return this.renderStartScreen(); 
+      if(this.skin.state.desiredstate !="desired_play") {
```
Comment (lental) [inline sdk/react/ooyalaSkinPanelRenderer.js]: uh oh! needs capital S in desiredState

Comment (wunnath) [inline sdk/react/ooyalaSkinPanelRenderer.js]: changed

**Comment by wunnath** [comment]
commited the changes based on the comments.

File: sdk/react/index.android.js [hunk 3]
```diff
@@ -43,6 +43,7 @@ var OoyalaSkin = React.createClass({
       cuePoints: [],
       promoUrl: '',
       hostedAtUrl: '',
+      desiredState:'desired_pause',
```
Comment (lental) [inline sdk/react/index.android.js]: sorry, can you make contstants in the Constants file?

Comment (wunnath) [inline sdk/react/index.android.js]: added to constants :)

**Pull request merged**